tests/e2e/utils/reset.js

```diff
@@ -32,21 +32,37 @@ import {
 
 /**
  * Resets Site Kit using a utility plugin.
+ *
+ * @since 1.0.0
+ * @since n.e.x.t Option to reset persistent options.
+ *
+ * @param {Object} options Reset Site kit Options.
  */
-export async function resetSiteKit() {
+export async function resetSiteKit( options = { persistent: false } ) {
```

Comment:
Let's do this slightly differently for consistency with how we do this most everywhere else which is specifying the default per-key rather than all or nothing.
```suggestion
export async function resetSiteKit( { persistent = false } = {} ) {
```